### PR TITLE
Bump postgres versions used in CI

### PR DESCRIPTION
**Pull request by svenklemm**
Change CI tests to run against pg 11.12, 12.7 and 13.3.

### CODE REVIEW

File: scripts/gh_matrix_builder.py [hunk 1]
```diff
@@ -135,6 +135,12 @@ def macos_config(overrides):
   }
   m["include"].append(build_debug_config(pg12_debug_earliest))
 
+  # add debug test for first supported PG12 version
```
Comment (gayyappan) [inline scripts/gh_matrix_builder.py:138]:
```suggestion
  # add debug test for first supported PG13 version
```

File: test/pgtest/CMakeLists.txt [hunk 2]
```diff
@@ -25,6 +25,7 @@ file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/testtablespace)
 # Tests to ignore
 set(PG_IGNORE_TESTS
   advisory_lock
+  event_trigger
   foreign_data
```
Comment (mkindahl) [inline test/pgtest/CMakeLists.txt:29]: Accidental addition? Does not look related.

Comment (svenklemm) [reply test/pgtest/CMakeLists.txt:29]: This is actually related since latest PG versions print a list of all defined event triggers in the event_trigger test making the output different when timescaledb is installed since we create our own event triggers.

https://github.com/postgres/postgres/commit/f3c4537876843bea9a9d35d9643d347750961848

Comment (mkindahl) [reply test/pgtest/CMakeLists.txt:29]: Ah, ok.

**Review by mkindahl — APPROVED** [review]

**Review by pmwkaa — APPROVED** [review]

**Review by afiskon — APPROVED** [review]
